lib/ae_mdw/aex141.ex

```diff
@@ -126,6 +149,16 @@ defmodule AeMdw.Aex141 do
     end
   end
 
+  defp deserialize_cursor(Model.NftTemplate, cursor_bin64) do
+    with {:ok, cursor_bin} <- Base.decode64(cursor_bin64),
+         {<<_pk::256>>, template_id} = cursor when is_integer(template_id) <-
+           :erlang.binary_to_term(cursor_bin) do
```

Comment:
We should add another check for the template_id value type and length, it could be any erlang term that the use can build, including functions and large structures

Reply:
There is a `when is_integer(template_id)`

Reply:
Thought of something else, my bad!